firefox-ios/Client/Coordinators/SettingsCoordinator.swift

```diff
@@ -222,6 +222,12 @@ class SettingsCoordinator: BaseCoordinator,
 
     // MARK: PrivacySettingsDelegate
 
+    func pressedAddressAutofill() {
+        let viewModel = AddressAutofillSettingsViewModel()
+        let viewController = AddressAutofillSettingsViewController(addressAutofillViewModel: viewModel)
+        router.push(viewController)
```

Comment:
curious - why do we not use the `findAndHandle` similar to `pressedCreditCard`

Reply:
Credit card is going through a different flow, it goes through a passcode page then the credit card page

Reply:
thanks for clarifying :)!